### ISSUE TITLE
run exe file with args from StreamingAssets?

### ISSUE THREAD

**Issue by davoodkharmanzar** [josh4364/IL2cppStartProcess#5]
hello and thanks for this new :]

how can i run an exe file with their args that exists on StreamingAssets folder?

StartExternalProcess.Start(Application.streamingAssetsPath, Application.streamingAssetsPath + "/EXEFILE" + args); ??

**Comment by davoodkharmanzar** [comment]
hello,
anybody there?

**Comment by zabszk** [comment]
Just separate arguments using spaces. "path/to/executable.exe arg1 arg2 args".

**Comment by davoodkharmanzar** [comment]
> Just separate arguments using spaces. "path/to/executable.exe arg1 arg2 args".

you mean something like this?

ExternalProcess.Start(System.IO.Path.Combine(Application.streamingAssetsPath, "EXE ") + $" ARGS");

**Comment by josh4364** [comment]
Yeah something like that, just create a single path string, with the args inline
